### ISSUE TITLE
SimpleAccessorNameNotation: ignore override method

### ISSUE THREAD

**Issue by rnveach** [sevntu-checkstyle/sevntu.checkstyle#777]
Split from https://github.com/sevntu-checkstyle/sevntu.checkstyle/issues/166 ,

````
$ cat TestClass.java
package com.github.sevntu.checkstyle.checks.coding;

public class InputSimpleAccessorNameNotationCheckOverride extends ParentClass {
    @Override
    public String getOne() {
        return two;
    }

    @java.lang.Override
    public String getTwo() {
        return one;
    }
}

abstract class ParentClass {
    protected String one;
    protected String two;

    public abstract String getOne();

    public abstract String getTwo();
}


$ cat TestConfig.xml
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
          "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
          "https://checkstyle.org/dtds/configuration_1_3.dtd">

<module name="Checker">
    <property name="charset" value="UTF-8"/>

    <module name="TreeWalker">
<module name="com.github.sevntu.checkstyle.checks.coding.SimpleAccessorNameNotationCheck" />
    </module>
</module>

$ java -jar checkstyle-8.23-sevntu-1.35.0-all.jar -c TestConfig.xml TestClass.java
Starting audit...
[ERROR] TestClass.java:4:5: Unexpected getter name. [SimpleAccessorNameNotation]
[ERROR] TestClass.java:9:5: Unexpected getter name. [SimpleAccessorNameNotation]
Audit done.
Checkstyle ends with 2 errors.
````

Expected no violations since method is overridden and have no choice on name of method.

**Comment by rnveach** [comment]
Once released, a few suppressions in checkstyle can be removed from this.

**Comment by romani** [comment]
Fix is merged
